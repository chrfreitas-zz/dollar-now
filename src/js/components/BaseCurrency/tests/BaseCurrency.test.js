import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

import BaseCurrency from './../BaseCurrency';

beforeAll(() => {
    Enzyme.configure({ adapter: new Adapter() })
});

describe('BaseCurrency', () => {
    it('should return true', () => {
        const BaseCurrencyTest = shallow(<BaseCurrency/>);
        expect(BaseCurrencyTest.exists()).toBeTruthy();
    });

    it('should to match with snapshot', () => {
        const BaseCurrencyTest = renderer.create(<BaseCurrency>1 USD</BaseCurrency>).toJSON();
        expect(BaseCurrencyTest).toMatchSnapshot();
    });
})


