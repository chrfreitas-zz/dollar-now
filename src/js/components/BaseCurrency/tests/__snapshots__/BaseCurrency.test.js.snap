// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseCurrency should to match with snapshot 1`] = `
<button
  className="sc-bwzfXH dcEJXz"
>
  1 USD
</button>
`;
